src/testInteg/sam.test.ts

```diff
@@ -345,7 +345,9 @@ async function stopDebugger(logMsg: string | undefined): Promise<void> {
 async function activateExtensions(): Promise<void> {
     console.log('Activating extensions...')
     await vscodeUtils.activateExtension(VSCODE_EXTENSION_ID.python, false)
-    await vscodeUtils.activateExtension(VSCODE_EXTENSION_ID.go, false)
+    // TODO: Must be reactivated when go tests are enabled above.
+    // Caveat: v0.40.1 of the go extension breaks this line (see changelog for this version)
```

Comment:
Thanks for noting that. Would be very much worth mentioning the details in the commit message.

https://github.com/golang/vscode-go/releases/tag/v0.40.1

> We adjusted the extension activation behavior. The extension will not be activated until Go files are open or the extension's commands are invoked.